km_api/know_me/views.py

```diff
@@ -1,4 +1,5 @@
-"""Views for the ``know_me`` module.
```

Comment:
Somehow the order of the views here got mixed up. If you could put them in alphabetical order that would be great.